chapter_convolutional-neural-networks/batch-norm.md

```diff
@@ -9,7 +9,7 @@
 
 ## 批量归一化层
 
-对全连接层和卷积层做批量归一化的方法稍有不同。下面我们将分别介绍这两种情况下的批量归一化。
+对全连接层和卷积层做批量归一化的方法稍有不同，下面我们将分别介绍这两种情况下的批量归一化。
 
```

Comment:
We tried to keep "one sentence, one meaning". For technical discussions, short sentences are easier for readers.